nyaa/backend.py

```diff
@@ -211,6 +211,10 @@ def handle_torrent_upload(upload_form, uploading_user=None, fromAPI=False):
     # To do, automatically mark trusted if user is trusted unless user specifies otherwise
     torrent.trusted = upload_form.is_trusted.data if can_mark_trusted else False
 
+    # Only allow mods to upload locked torrents
+    can_mark_locked = uploading_user and uploading_user.is_moderator
```

Comment:
make it possible for trusted users to upload their torrents with comments locked?
also s/locked/comment locked/